pom.xml

```diff
@@ -158,7 +158,7 @@
         <hive.archive.download.skip>false</hive.archive.download.skip>
         <httpclient.version>4.5.14</httpclient.version>
         <httpcore.version>4.4.16</httpcore.version>
-        <iceberg.version>1.3.1</iceberg.version>
+        <iceberg.version>1.4.0</iceberg.version>
```

Comment:
let's do it independently

Reply:
Do you mean submitting a new PR or specifying it in spark-3.5 profile?

Reply:
I mean upgrading Iceberg in separated PR

Reply:
Got it, thanks.

Reply:
I have specified the iceberg version in spark-3.5 profile and will send a new PR to upgrade the default iceberg version